### PR TITLE
Document new Webswing options

### PR DESCRIPTION
**Pull request by psiinon**
Can review now but dont merge until the stable docker image is regenerated.

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: site/content/docs/docker/webswing.md [hunk 1]
```diff
@@ -26,6 +26,38 @@ The ZAP UI acts in just the same way as it does when running 'natively'.
...
+If you restart the docker container using the same mapped drive then ZAP will automatically import the private ZAP Root CA certificate 
+so you will not need to import it into your browser again.
+
+
 Note that you will need to visit http://localhost:8080/zap _before_ you can use ZAP as a proxy - ZAP will only start to run when you visit that URL.
+
+## ZAP Command Line Options
+
+When you run ZAP using Webswing then the following ZAP Command Line Options will be used:
+
```
Comment (kingthorin) [inline site/content/docs/docker/webswing.md:56]: I think you missed pasting a line here?

Comment (psiinon) [reply site/content/docs/docker/webswing.md:56]: Er, no, its supposed to refer to the following lines :/

Comment (kingthorin) [reply site/content/docs/docker/webswing.md:56]: Oh I guess I was anticipating the "..the following default options are used...." line.... okay no problem

**Review by kingthorin — APPROVED** [review]

**Comment by psiinon** [comment]
Note that the docker stable image has now been updated :)

**Review by thc202 — APPROVED** [review]

**Pull request merged**